sdk/src/types/block/parent.rs

```diff
@@ -55,11 +54,21 @@ impl<const MIN: u8, const MAX: u8> Parents<MIN, MAX> {
         ))
     }
 
+    /// Returns the unique, ordered set of parents.
+    pub fn to_set(&self) -> BTreeSet<BlockId> {
+        self.0.iter().copied().collect()
+    }
+
     /// Returns the number of parents.
     pub fn len(&self) -> usize {
         self.0.len()
     }
 
+    /// Returns whether the parents list is empty.
+    pub fn is_empty(&self) -> bool {
+        self.len() == 0
```

Comment:
```suggestion
        self.0.is_empty()
```

Reply:
I'd rather use the type's impl, just because it's more flexible.

Reply:
okay